### ISSUE TITLE
API Groups follow-up

### ISSUE THREAD

**Issue by mfojtik** [openshift/origin#13412]
This issue attemps to track all possible breakages we might/will encounter after https://github.com/openshift/origin/pull/12986 lands.

- [x] generate clientsets
- [x] Make sure all controller SA have rules updated to work with API groups
- [x] Audit all admission controllers that they deal with grouped/non-grouped resources equally and prepare migration if they don't
- [x] Fix ImagePolicy admission to deal with both legacy and grouped resources

**Comment by mfojtik** [comment]
@sttts @liggitt @deads2k @soltysh an follow-up issue tracker for the API groups.

**Comment by soltysh** [comment]
This is super important and needs to happen within 1.6 timeframe.

**Comment by liggitt** [comment]
(and before any production pushes from origin)

**Comment by deads2k** [comment]
Needs to be done before the next push to online.

**Comment by mfojtik** [comment]
Closing this as all follow ups seems to be adressed now.
